client/my-sites/site-settings/section-writing.jsx

```diff
@@ -20,9 +20,9 @@ module.exports = React.createClass( {
 	render: function() {
 
 		return (
-			<Card className="writing-settings">
-				<WritingForm site={ this.props.site } />
-			</Card>
+
+			<WritingForm site={ this.props.site } />
```

Comment:
Also, am I missing something big here? Does this component have more-to-come because it looks like it doesn't do anything but print out a debug statement.

If it doesn't have more coming, can we not replace it altogether with the `<WritingForm />`